src/react-components/room/SafariMicModal.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Modal } from "../modal/Modal";
+import { Button } from "../input/Button";
+import styles from "./SafariMicModal.scss";
+
+export function SafariMicModal() {
+  return (
+    <Modal title="Microphone Access Required">
+      <div className={styles.safariMicModal}>
+        <p>Hubs requires microphone permissions in Safari.</p>
```

Comment:
(Not urgent but) might be nice to add a screenshot of the mic permissions dialog. Not sure how often people run into this or if they get confused by it. Just thinking out loud...